week08/movie/src/pages/MainPage.js

```diff
@@ -19,27 +22,36 @@ const MovieSet = styled.div`
 const SetText = styled.h3`
   margin: 8px 15px 0px;
   margin-bottom: -8px;
+  color: ${(props) => (props.theme.mode === "dark" ? "#f29b3e" : "#ffffff")};
```

Comment:
Header 부분에서도 설명드렸지만, Theme.js에서 darkmode와 lightmode에 사용할 background 색상을 정의해주었으므로, 각 컴포넌트에서 색상을 가져다가 쓸 때는 이미 정의해둔 객체 프로퍼티를 가져다가 쓰면 됩니다.

```suggestion
  background: ${(props) => props.theme.backgroundColor};
```